sktime/transformers/tests/test_Rocket.py

```diff
@@ -0,0 +1,30 @@
+import numpy as np
+
+from sktime.datasets import load_gunpoint
+from sktime.transformers.rocket import Rocket
+from sklearn.linear_model import RidgeClassifierCV
+
+def test_rocket_on_gunpoint():
```

Comment:
Thanks for adding the test! 
 * It'd be great to have some `assert` statements in the test (e.g. using the `np.testing` modules) checking dimension/shapes of output and so on, 
* I wouldn't do anything too computationally expensive in unit tests, so no need to run grid-search cv or many kernels.

Reply:
I'll add some checks.

I think the `test_rocket_on_gunpoint(...)` function should be ok in terms of runtime: it runs in about 3 seconds on my 5-year-old laptop (if this is too much, I can reduce the number of kernels).

`RidgeClassifierCV` by default uses generalized cross-validation (a kind of estimated loocv), so the time cost of performing the cross-validation should be basically imperceptible, at least on small datasets.  Here the classifier runtime (training + test) is about 70 milliseconds including cross-validation.